### PR TITLE
CHEMH-212, CHEMH-219: removing animations and lines

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Removing the animation from the main navigation for both mobile and desktop
- Removed ghost line on the mobile nav.
- Fixed arrow direction in mobile menu 

# Review By (Date)
- Soon

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to to main site in desktop and then mobile
4. Verify the hover and active states are no longer animated.
5. Verify the hamburger is also not animated.
6. Use the tab key to navigate the mobile menu. Make sure the arrow rotates when the menu closes but keeps the red to indicate the keyboard focus. 

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- CHEMH-212](https://stanfordits.atlassian.net/browse/CHEMH-212)
[- CHEMH-219](https://stanfordits.atlassian.net/browse/CHEMH-219)
[- CHEMH-208](https://stanfordits.atlassian.net/browse/CHEMH-208)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by jdwjdwjdw — COMMENTED** [review]
Should the dropdown icons rotate on hover? I'd assume they would only rotate once the dropdown menu has been opened, similar to our sites platform.

**Comment by jenbreese** [comment]
The designer had me remove all the animations so I would think no.

**Comment by jenbreese** [comment]
Note from designer: 
I think it could change color, but I wouldn’t swap direction until it was clicked

**Comment by jenbreese** [comment]
I left in the `focus` for our keyboard users.

**Comment by jenbreese** [comment]
@jdwjdwjdw I fixed up the `hover` issue.

File: src/scss/components/main-navigation/_main-navigation.scss [hunk 1]
```diff
@@ -122,8 +126,7 @@
             background: transparent url("../../assets/svg/icon_expand_more_white.svg") no-repeat center / 50%;
           }
 
-          &:focus,
-          &:hover {
+          &:focus {
```
Comment (jenbreese) [inline src/scss/components/main-navigation/_main-navigation.scss:133]: @jdwjdwjdw I removed the hover here.

**Review by jdwjdwjdw — APPROVED** [review]
Looks great! Thanks @jenbreese 🚀

**Pull request merged**